src/vm-vcpu/src/vcpu/mod.rs

```diff
@@ -395,7 +397,7 @@ impl KvmVcpu {
                             interrupted_by_signal = true;
                         }
                         _ => {
-                            eprintln!("Emulation error: {}", e);
+                            debug!("Emulation error: {}", e);
```

Comment:
Hmm shouldn't we return an error from the function when an emulation error occurs?

Reply:
I am not really sure. We should probably. I don't know if emulation errors should be considered fatal, in which case we need to exit.

Reply:
I think we should return an error and relax this later if it turns out too restrictive. No super strong preference, but it sounds like the right thing to IMO.